internal/i18n/en_translations.go

```diff
@@ -211,4 +211,12 @@ var (
 	MsgTokenTransferFailed         = ffm("FF10291", "Token transfer with ID '%s' failed. Please check the FireFly logs for more information")
 	MsgFieldNotSpecified           = ffm("FF10292", "Field '%s' must be specified", 400)
 	MsgTokenPoolNotConfirmed       = ffm("FF10293", "Token pool is not yet confirmed")
+	MsgHistogramStartTimeParam     = ffm("FF10294", "Start time of the data to be fetched")
+	MsgHistogramEndTimeParam       = ffm("FF10295", "End time of the data to be fetched")
+	MsgHistogramBucketsParam       = ffm("FF10296", "Number of buckets between start time and end time")
+	MsgHistogramCollectionParam    = ffm("FF10297", "Collection to fetch")
+	MsgInvalidNumberOfIntervals    = ffm("FF10298", "Number of time intervals must be between %d and %d")
+	MsgInvalidChartNumberParam     = ffm("FF10299", "Invalid %s. Must be a number.", 400)
+	MsgHistogramInvalidTimes       = ffm("FF10300", "Start time must be before end time")
+	MsgUnsupportedCollection       = ffm("FF10301", "%s collection is not supported")
```

Comment:
I think all of these last 4 can be HTTP 400

Reply:
done